### PR TITLE
Feature/cc 12343

### PR DESCRIPTION
**Pull request by rs-sdet**
These are changes related to cc-12343

### CODE REVIEW

File: src/test/resources/features/IFramePrectice.feature [hunk 1]
```diff
@@ -12,8 +12,6 @@ Feature: iFrame handling
     When user enters the value in input iframe
     Then user should be seeing entered value
 
-  @iframe_handling2
-  Scenario: iFrame handling
-    Given user launches the browser with new url
-    When user enters the value in input iframe
-    Then user should be seeing entered value
+    @feature_handling
```
Comment (Riyaz-commits) [inline src/test/resources/features/IFramePrectice.feature:15]: Hi @ritusolanki8  ,

Can we know why are you removed the test case

File: src/test/resources/features/IFramePrectice.feature [hunk 2]
```diff
@@ -12,8 +12,6 @@ Feature: iFrame handling
     When user enters the value in input iframe
     Then user should be seeing entered value
 
-  @iframe_handling2
```
Comment (Riyaz-commits) [inline src/test/resources/features/IFramePrectice.feature:15]: please add back this scenario thanks

**Review by Riyaz-commits — CHANGES_REQUESTED** [review]
PLease address the changes

File: src/test/resources/features/IFramePrectice.feature (hunk 1 above)
Comment (rs-sdet) [reply src/test/resources/features/IFramePrectice.feature:15]: Changes have been addressed, please review..

**Review by Riyaz-commits — APPROVED** [review]
LGTM!

**Pull request merged**